packages/RNTester/js/components/RNTesterBlock.js

```diff
@@ -10,108 +10,94 @@
 
 'use strict';
 
-const React = require('react');
-
-const {StyleSheet, Text, View} = require('react-native');
-import {RNTesterThemeContext} from './RNTesterTheme';
-
 type Props = $ReadOnly<{|
   children?: React.Node,
   title?: ?string,
   description?: ?string,
   isDisabled: ?boolean,
 |}>;
 
-type State = {|
-  description: ?string,
-|};
-
-class RNTesterBlock extends React.Component<Props, State> {
-  state: State = {description: null};
-
-  render(): React.Node {
-    const description = this.props.description ? (
-      <RNTesterThemeContext.Consumer>
-        {(theme) => {
-          return (
-            <Text style={[styles.descriptionText, {color: theme.LabelColor}]}>
-              {this.props.description}
-            </Text>
-          );
-        }}
-      </RNTesterThemeContext.Consumer>
-    ) : null;
+import React, {useState} from 'react';
+import {RNTesterThemeContext} from './RNTesterTheme';
+import {StyleSheet, Text, View} from 'react-native';
 
-    return (
+/** functional component for generating example blocks */
+const RNTesterBlock = ({description, isDisabled, title, children}: Props) => {
+  let descComponent = null;
+  isDisabled = [false, true, false, false, false, false, true, false, true][
```

Comment:
Can you please explain tp me why do we need to do this?

Reply:
this shouldn't have been committed :disappointed: